wg-security-audit/RFP.md

```diff
@@ -96,4 +96,11 @@ The audit should result in the following artifacts, which will be made public af
 
 ## Q & A
 
-This section intentionally left empty until the question period opens.
+| # | Question | Answer |
+|---|----------|--------|
+| 1 | The RFP says that any area included in the out of scope section of the k8s bug bounty programme is not in-scope of this review.  There are some areas which are out of scope of the bug bounty which would appear to be relatively core to k8s, for example Kubernetes on Windows. Can we have 100% confirmation that these areas are out of scope? | Yes. If you encounter a vulnerability in Kubernetes' use of an out-of-scope element, like etcd or the container network interface (to Calico, Weave, Flannel, ...), that is in scope. If you encounter a direct vulnerability in a third-party component during the audit, you can report it to them.  |
+| 2 | On the subject of target Distribution and configuration option review:<br> The RFP mentions an "audited reference architecture".<br> -	Is the expectation that this will be based on a specific k8s install mechanism (e.g. kubeadm)? <br> -	On a related note is it expected that High Availability configurations (e.g. multiple control plane nodes) should be included.<br> -	The assessment mentions Networking as a focus area.  Should a specific set of network plugins (e.g. weave, calico, flannel) be considered as in-scope or are all areas outside of the core Kubernetes code for this out of scope.<br> -	Where features of Kubernetes have been deprecated but not removed in 1.12, should they be considered in-scope or not? | 1. No, we are interested in the final topology -- the installation mechanism, as well as its default configuration, is tangental. The purpose is to contextualise the findings.<br>2. High-availability configurations should be included. For confinement of level of effort, vendor could create one single-master configuration and one high-availability configuration.<br>3. All plugins are out of scope per the bug bounty scope -- for clarification regarding the interface to plug-ins, please see the previous question.<br> 4. Deprecated features should be considered out of scope" |
```

Comment:
There seems to be a trailing double-quote at the end of point 2.

Reply:
Thanks!